src/modules/profile/pages/profileSettings/components/CreatorNotifications.tsx

```diff
@@ -0,0 +1,270 @@
+import { useMutation } from '@apollo/client'
+import { HStack, Select, Switch, VStack } from '@chakra-ui/react'
+import { useToast } from '@chakra-ui/react'
+import { useEffect, useState } from 'react'
+import { PiStar } from 'react-icons/pi'
+
+import { ImageWithReload } from '@/components/ui'
+import { MUTATION_UPDATE_CREATOR_NOTIFICATIONS_SETTINGS } from '@/modules/profile/graphql/mutations/creatorNotificationMutation'
+import { CardLayout } from '@/shared/components/layouts'
+import { Body } from '@/shared/components/typography'
+import { CreatorNotificationSettings } from '@/types'
+
+import { HorizontalFormField } from '../common/HorizontalFormField'
+
+enum NotificationType {
+  PROJECT_SUMMARY = 'creator.projectSummary',
+  GOAL_REACHED = 'creator.goalReached',
+  SALE_MADE = 'creator.saleMade',
+  CONTRIBUTION_RECEIVED = 'creator.contributionReceived',
+}
+
+enum ConfigName {
+  FREQUENCY = 'frequency',
+  IS_ENABLED = 'is_enabled',
+  THRESHOLD = 'threshold',
+}
+
+export const CreatorNotifications = ({
+  creatorNotificationSettings,
+}: {
+  creatorNotificationSettings: CreatorNotificationSettings[]
+}) => {
+  const [settings, setSettings] = useState<CreatorNotificationSettings[]>(creatorNotificationSettings)
+
+  useEffect(() => {
+    setSettings(creatorNotificationSettings)
+  }, [creatorNotificationSettings])
+
+  const [updateNotificationSetting] = useMutation(MUTATION_UPDATE_CREATOR_NOTIFICATIONS_SETTINGS)
+  const toast = useToast()
```

Comment:
We use, useNotification() for all of the toast, through out the platform. The timings and position are already preconfigured there
I've updated it so that it needs minimum parameters as possible. 

Let's use that for consistency.

```suggestion
  const toast = useNotification()
```